src/js/components/Header/VolumeControl.react.js

```diff
@@ -32,7 +32,7 @@ export default class VolumeControl extends Component {
     }
 
     getVolumeIcon(volume, muted) {
-        return muted || volume === 0 ? 'volume-off' : volume > 0.5 ? 'volume-up' : 'volume-down';
+        return muted || volume === 0 ? 'volume-off' : Math.pow(volume, 1 / 4) > 0.5 ? 'volume-up' : 'volume-down';
```

Comment:
Yea, I think that makes sense. We've essentially just changed the component to an exponential slider, and any outside observers can just use the exponential value. They shouldn't have a need for the linear value.

However, I _could_ see another volume slider needing the linear volume value. eg: if we put a slider in the tray. But we can address that concern when we get to it.